examples_utils/benchmarks/command_utils.py

```diff
@@ -153,9 +153,10 @@ def formulate_benchmark_command(
     else:
         py_name = "python"
     called_file = cmd_parts[cmd_parts.index(py_name) + 1]
-
-    resolved_file = str(Path(called_file).resolve())
-    cmd = cmd.replace(called_file, resolved_file)
+    # if the first argument is `-m` we are calling a module and shouldn't resolve it
+    if called_file != "-m":
```

Comment:
To be clear, cmd here is referring to the command in the yaml file, not the command used to call this benchmarking module. 

Do you intend to have modules called in the yaml file commands?

Reply:
So this was a problem picked up by the test, the notebook "benchmarks" don't define a "cmd" it gets defined programmatically in `process_notebook_to_command` that command is of the format `python -m examples_utils.benchmarks.notebook_utils ...` [defined here](https://github.com/graphcore/examples-utils/pull/40/files#diff-47e2294c346c9d48f9d81b28e3bfe3884b63990dcfbfa5cd77df59e21124ed5dR344) then argument 1 is not a file that needs to be sanitised.